provisioning/device/samples/readme.md

```diff
@@ -0,0 +1,101 @@
+# Provisioning Device Client Sample - Microsoft Azure IoT SDK for .NET
+
+## Overview
+
+_This documentation is preliminary and subject to change._
+
+This folder contains samples demonstrating the steps required to dynamically associate devices with IoT Hubs using the Microsoft Azure IoT Device Provisioning Service.
```

Comment:
Nit: "IoT Hubs" should be "IoT hubs" (lowercase h). Yes I would like to have Words with the person who made the capitalization rules.